src/desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -76,7 +76,7 @@ module.exports = class ArtworkAuctionView extends Backbone.View
         quantity: 1
         user: loggedInUser
       .then (data) ->
-        order = data?.createOrderWithArtwork?.orderOrError?.order
+        order = data?.ecommerceCreateOrderWithArtwork?.orderOrError?.order
```

Comment:
@orta - curious where recent discussions shook out around prefixing service names to stitched queries? I know that's typically understood to be a bad habit (or dogma, depending on perspective 😄) but I can also see some benefit in terms of specifically-targeted future refactors.

Reply:
Last time we talked about it we ended up preferring the genre of the type - I think @alloy had a pretty reasonable opinion that prefixes note the domain of the API but not the specifics in case we find more places where there services overlap

Reply:
I think I preferred it because you didn't need to know the service names, just the general area for the schema when you were working from a client side. Right now, exchange = ecommerce is obvious to us because it's fresh in our minds, but eventually (hopefully) it won't

Reply:
Makes sense 👍

Reply:
👍 My argument can be simplified to prefixing with service name is leaking implementation details, rather than semantic details.